src/org/janelia/mipav/plugins/worm/untwisting/PlugInDialogVolumeRenderDualJanelia.java

```diff
@@ -2255,30 +2337,72 @@ private void init() {
 		dialogGUI.getContentPane().add(panel1, BorderLayout.NORTH);
 		dialogGUI.getContentPane().add(panel2, BorderLayout.SOUTH);
 
-		lutPanel = new JPanel(new GridLayout(2,1));
+		lutPanel = new JPanel(new GridLayout(2, 1));
 		lutTab = new JTabbedPane();
 		lutTab.addChangeListener(this);
 		lutPanel.add(lutTab, BorderLayout.CENTER);
-		opacityPanel = new JPanel( new BorderLayout() );
+		opacityPanel = new JPanel(new BorderLayout());
 		opacityTab = new JTabbedPane();
 		opacityTab.addChangeListener(this);
 		opacityPanel.add(opacityTab, BorderLayout.CENTER);
+		clipPanel = new JPanel(new BorderLayout());
+
+		// Create a panel for toggling between accurate and 3-color modes(not working
+		// currently)
+		accuratePanel = new JPanel(new BorderLayout());
+		accuratePanel.setPreferredSize(new Dimension(400, 400));
+		accuratePanel.setMaximumSize(new Dimension(400, 400));
+		JPanel buttonPanel = new JPanel();
 		
-		clipPanel = new JPanel( new BorderLayout() );
+		// Initialize a toggle button for switching modes
+		accurateModeButton = new JToggleButton("Accurate Mode");
+		accurateModeChanged(true);
+		accurateModeButton.setPreferredSize(new Dimension(200, 50));
+
+		// Add an action response to the button
+		accurateModeButton.addActionListener(new ActionListener() {
+			@Override
+			public void actionPerformed(ActionEvent e) {
+				JToggleButton toggleButton = (JToggleButton) e.getSource();
+				boolean isSelected = toggleButton.isSelected();
+				activeImage.voiManager.toggleAccurateMode();
+			}
+		});
+
+		// Add the toggle button to the button panel and then add the panel to the
+		// accurate mode panel
+		buttonPanel.add(accurateModeButton);
+		accuratePanel.add(buttonPanel, BorderLayout.NORTH);
+
+		chartPanel = new SelectionChartPanel(Arrays.asList(1.0f, 2.0f, 3.0f), "Chart", this);
+		
+		// Created a fakePanel to split the panel
+		JPanel fakePanel = new JPanel();
+		fakePanel.setPreferredSize(new Dimension(100,500));	
```

Comment:
Do we need this?

```suggestion
```
```suggestion
		fakePanel.setPreferredSize(new Dimension(100,500));	
```

Reply:
yes, we do need it

Reply:
- [ ] added //comment for the changes if remove the line